lib/ecmascript.ts

```diff
@@ -3412,7 +3434,7 @@ export function BalanceDurationRelative(
   if (sign === 0) return { years, months, weeks, days };
 
   let calendar;
-  let relativeTo: Temporal.PlainDate;
+  let relativeTo = undefined as Temporal.PlainDate | undefined;
```

Comment:
I'm not very familiar with TS best practices, but should this be something like `let relativeTo: Temporal.PlainDate? = undefined;`?